packages/ethereum-light-client/src/types/wrappers.rs

```diff
@@ -0,0 +1,162 @@
+use alloy_primitives::{aliases::B32, Bloom, Bytes, FixedBytes, B256};
```

Comment:
A lot of these wrapped types are added to enable tree_hash_root in special corner cases where tree_hash_derive did not work properly for some reason. I think it might be possible to have way fewer of these, but I didn't manage in a reasonable time and didn't want to spend too much time on it right now. Tracking in https://github.com/cosmos/solidity-ibc-eureka/issues/147